sql/split.go

```diff
@@ -124,7 +127,18 @@ func (n *splitNode) Start() error {
 		return err
 	}
 	n.key = keys.MakeRowSentinelKey(key)
-	return n.p.execCfg.DB.AdminSplit(n.key)
+
+	// We retry a few times if we hit an error. One instance of expected errors
+	// happens after a table creation, where AdminSplit will sometimes fail with
+	// "conflict updating range descriptors".
+	var splitErr error
+	for r := retry.Start(retry.Options{MaxRetries: maxSplitRetries}); r.Next(); {
+		splitErr = n.p.execCfg.DB.AdminSplit(n.key)
+		if splitErr == nil {
+			return nil
```

Comment:
TFTR! I don't see the problem with retrying on all errors, given that we're only doing a few retries. If it's a persistent error, we will fail soon enough anyway, and if it's transient and it goes away, all the better..